cmd/apptainer.lima

```diff
@@ -7,4 +7,4 @@ if [ "$(limactl ls -q "$LIMA_INSTANCE" 2>/dev/null)" != "$LIMA_INSTANCE" ]; then
   exit 1
 fi
 export LIMA_INSTANCE
-exec lima APPTAINER_HOME="$HOME" apptainer "$@"
+exec lima APPTAINER_BINDPATH="$HOME" apptainer "$@"
```

Comment:
It would be better to not assume that APPTAINER_BINDPATH isn't already set in the environment.  Extend it if it is already set, with a comma separator.

Even worse, there's the possibility that SINGULARITY_BINDPATH is set as well, which apptainer would honor (with a deprecation warning) if you hadn't set APPTAINER_BINDPATH here, but setting only APPTAINER_BINDPATH here overrides that (with another warning).  So the safest thing to do is something like this:
```
if [ -n "$APPTAINER_BINDPATH" ]; then
    SINGULARITY_BINDPATH="$APPTAINER_BINDPATH"
fi
if [ -n "$SINGULARITY_BINDPATH" ]; then
  SINGULARITY_BINDPATH="$SINGULARITY_BINDPATH,"
fi
SINGULARITY_BINDPATH="$SINGULARITY_BINDPATH$HOME"
exec lima APPTAINER_BINDPATH="$SINGULARITY_BINDPATH" apptainer "$@"
```
I deliberately didn't export SINGULARITY_BINDPATH because it isn't necessary.  On the other hand if it was previously exported, it won't hurt -- apptainer will see they're both the same and let it go.  I am assuming that lima passes all of its environment to apptainer except for anything overridden on the command line..

Reply:
The environment is being filtered through the ssh command, which has `ssh_config` (`SendEnv`) and `sshd_config` (`AcceptEnv`) setting. In this wrapper script, we are _explicitly_ setting it in the wrapper script (i.e. for bash) being invoked...

`DEBU[0000] executing ssh (may take a long)): [/usr/bin/ssh -F /dev/null -o IdentityFile="/home/anders/.lima/_config/user" -o IdentityFile="/home/anders/.ssh/id_rsa" -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -o NoHostAuthenticationForLocalhost=yes -o GSSAPIAuthentication=no -o PreferredAuthentications=publickey -o Compression=no -o BatchMode=yes -o IdentitiesOnly=yes -o Ciphers="^aes128-gcm@openssh.com,aes256-gcm@openssh.com" -o User=anders -o ControlMaster=auto -o ControlPath="/home/anders/.lima/apptainer/ssh.sock" -o ControlPersist=5m -t -o SendEnv="COLORTERM" -q -p 33473 127.0.0.1 -- cd /home/anders/lima || cd /home/anders ; exec "$SHELL" --login -c 'APPTAINER_BINDPATH=/home/anders apptainer version']`

If we think that it is going to be a common thing to install Singularity (Desktop?) on the Mac, then we can make more efforts in apptainer.lima and run-singularity scripts to allow for `singularity` and `SINGULARITY_BINDPATH` being present ?

* #1121

Reply:
The code snippet didn't work, partly due to unset variables (`-u`) - but it had some other issues as well.

`WARNING: While bind mounting '/home/anders:/home/anders': destination is already in the mount point list`

Reply:
Fixed APPTAINER_BINDPATH, can add singularity support in the other PR (i.e. SINGULARITY_BINDPATH)